frontend/talentsearch/src/js/components/diversityEquityInclusion/DiversityEquityInclusionForm.tsx

```diff
@@ -27,15 +59,134 @@ export const DiversityEquityInclusionForm: React.FC = () => {
+            "Description of how the Government of Canada uses employment equity categories in hiring.",
+        })}
+      </p>
+      <p>
+        {intl.formatMessage({
+          defaultMessage:
+            'These four groups are "women, Aboriginal peoples, persons with disabilities, and members of visible minorities."',
+          description: "List of the employment equity categories",
+        })}
+      </p>
+      <div
+        data-h2-bg-color="b(lightgray)"
+        data-h2-radius="b(s)"
+        data-h2-padding="b(all, m)"
+      >
+        <p data-h2-margin="b(top, none)">
+          {intl.formatMessage(
+            {
+              defaultMessage:
+                "<b>This section is optional</b>. If you a member of one or more of these employment equity groups, and you do not wish to self identify on this platform, there is no obligation to do so. <b>Complete the form below if you meet both of these conditions</b>:",
+              description:
+                "Explanation that employment equity information is optional.",
+            },
+            { b: boldText },
+          )}
+        </p>
+        <ul>
+          <li>
+            {intl.formatMessage({
+              defaultMessage:
+                "You are a member of one or more of these employment equity groups.",
+              description:
+                "Instruction on when to fill out equity information, item one",
+            })}
+          </li>
+          <li>
+            {intl.formatMessage({
+              defaultMessage:
+                "You would like to be considered for opportunities addressed to  underrepresented groups.",
+              description:
+                "Instruction on when to fill out equity information, item two",
+            })}
+          </li>
+        </ul>
       </div>
+      <h2>
+        {intl.formatMessage({
+          defaultMessage: "How will this data be used?",
+          description:
+            "Heading for how employment equity information will be used.",
+        })}
+      </h2>
+      <ul>
+        <li>
+          {intl.formatMessage({
+            defaultMessage:
+              "This information will be shared with hiring managers.",
+            description:
+              "Explanation on how employment equity information will be used, item one",
+          })}
+        </li>
+        <li>
+          {intl.formatMessage({
+            defaultMessage:
+              "The data here will be used to match you to prioritized jobs.",
+            description:
+              "Explanation on how employment equity information will be used, item two",
+          })}
+        </li>
+        <li>
+          {intl.formatMessage({
+            defaultMessage:
+              "This data will be use in an anonymous form for statistical purposes.",
+            description:
+              "Explanation on how employment equity information will be used, item three",
+          })}
+        </li>
+      </ul>
+      <EquityOptions
```

Comment:
Seems like the bottom padding is lost in narrow viewports
![Peek 2022-06-07 14-53](https://user-images.githubusercontent.com/8978655/172462888-e56ad175-0436-44a9-940e-7c7991d88be0.gif)
.

Reply:
Great catch! Resolved with [add margin to bottom of equity form](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2989/commits/d9fac380d8ab4d30530db1c1f3a4f2ac5efcd7f9)